### ISSUE TITLE
Trying to Edit Users as Site Admin: Error reading from database

### ISSUE THREAD

**Issue by jchrom** [iomad/iomad#870]
Hi,

On IOMAD (33_STABLE)
As a Site Admin
When I go to IOMAD dashboard -> User Management -> Edit users

I get the following error:

Error reading from database

```
More information about this error
Debug info: ERROR: invalid input syntax for integer: "2,3"
LINE 1: SELECT id FROM mdl_user WHERE id!='-1' AND id!='2,3' AND del...
^
SELECT id FROM mdl_user WHERE id!='-1' AND id!='2,3' AND deleted <> 1 AND suspended = 0 AND id in (4,5,6)
[array (
)]
Error code: dmlreadexception
Stack trace:

    line 486 of /lib/dml/moodle_database.php: dml_read_exception thrown
    line 242 of /lib/dml/pgsql_native_moodle_database.php: call to moodle_database->query_end()
    line 824 of /lib/dml/pgsql_native_moodle_database.php: call to pgsql_native_moodle_database->query_end()
    line 1653 of /lib/dml/moodle_database.php: call to pgsql_native_moodle_database->get_fieldset_sql()
    line 545 of /blocks/iomad_company_admin/editusers.php: call to moodle_database->get_fieldset_select()
```

**Comment by erazorbg** [comment]
You can find a fix in #350 it's the same issue, just need to add a comma.

This bug can only be reproduced if you have more than 1 site administrator.

**Comment by turf212** [comment]
The following files are exploding on " " the $CFG->siteadmins when they shouldn't be doing that:
./blocks/iomad_company_admin/editusers.php ./local/report_users/index.php
./local/report_user_licenses/index.php
./local/report_user_license_allocations/index.php
./local/report_license_usage/index.php
./local/report_user_logins/index.php

I'll change the way the code is using that setting
